staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1621,9 +1621,10 @@ type KeyToPath struct {
 	// May not contain the path element '..'.
 	// May not start with the string '..'.
 	Path string `json:"path" protobuf:"bytes,2,opt,name=path"`
-	// Optional: mode bits to use on this file, must be a value between 0
-	// and 0777. If not specified, the volume defaultMode will be used.
-	// This might be in conflict with other options that affect the file
+	// Optional: mode bits to use on this file, must be a decimal equivalent of
+	// octal value between 0 and 0777.
+	// If not specified, the volume defaultMode
+	// will be used. This might be in conflict with other options that affect the file
```

Comment:
as per first comment 
```suggestion
	// Optional: mode bits used to set permissions on this file.
	// Must be an octal value between 0 and 0777.
	// If not specified, the volume defaultMode will be used.
	// This might be in conflict with other options that affect the file
```